_type-styles.scss

```diff
@@ -7,44 +7,44 @@
 /*
   Utility that automatically generates all of the type styles
   for a project. Relies on $type-styles and $font-stacks map variables existing in the following format:
+*/
 
-  $font-stacks: (
-    futura-bold: (
-      font-family: (Futura, 'Trebuchet MS', Arial, sans-serif),
-      font-weight: 700,
-      font-style: normal
-    ),
-    helvetica: (
-      font-family: ('Helvetica Neue', Helvetica, Arial, sans-serif;),
-      font-weight: normal,
-      font-style: normal
+$font-stacks: (
+  futura-bold: (
+    font-family: (Futura, 'Trebuchet MS', Arial, sans-serif),
+    font-weight: 700,
+    font-style: normal
+  ),
+  helvetica: (
+    font-family: ('Helvetica Neue', Helvetica, Arial, sans-serif),
+    font-weight: normal,
+    font-style: normal
+  )
+) !default;
+
+$type-styles: (
+  heading: (
+    stack: futura-bold,
+    line-height: 1,
+    text-transform: normal,
+    letter-spacing: 0,
+    sizes: (
+      default: 14
     )
-  );
-
-  $type-styles: (
-    heading: (
-      stack: futura-bold,
-      line-height: 1,
-      text-transform: normal,
-      letter-spacing: 0,
-      sizes: (
-        default: 14
-      )
-    ),
-
-    body: (
-      stack: helvetica,
-      line-height: 1.4,
-      text-transform: uppercase,
-      letter-spacing: 1.2,
-      font-smoothing: true,
-      sizes: (
-        default: 16,
-        medium: 24
-      )
+  ),
+
+  body: (
+    stack: helvetica,
+    line-height: 1.4,
+    text-transform: uppercase,
+    letter-spacing: 1.2,
+    font-smoothing: true,
```

Comment:
Let's remove the `letter-spacing` and `font-smoothing` defaults for the `body` style.